website/docs/components/checkbox/examples/checkbox_with_other_components.tsx

```diff
@@ -11,32 +12,32 @@ function noop(e) {
 const Demo = () => (
   <>
     {[0, 1, 2].map((item) => (
-      <div key={item}>
-        <Checkbox mb={3} label={`Note ${item + 1}`} />
+      <Flex key={item}>
+        <Checkbox mb={3} label={`Option ${item + 1}`} />
         <Hint
```

Comment:
Yeah, definitely, thank you! Wanted to leave it to guys 😁

Reply:
> but ideally this is not the best example of using tooltips with checkboxes... maybe better to replace i's with something else whatsoever?

I've seen such combinations in other products, so I decided to leave them as they are. By the way, I left two checkboxes for this example because six options seemed excessive.

Reply:
Iliya replaced `Hint` with `DescriptionTooltip` in the https://github.com/semrush/intergalactic/pull/1457, so I resolve this 😎